controllers/api/reservationRoutes.js

```diff
@@ -1,6 +1,6 @@
 const router = require('express').Router();
-const { Reservation, Location, Room } = require('../../models');
-const { withAuth } = require('../../utils/auth');
+const { Reservation, Location, Room, User } = require('../../models');
```

Comment:
Will need to change from
withAuth
to
{ withAuth}
once merged with changes from "justin" branch